python/cugraph/tests/test_betweenness_centrality.py

```diff
@@ -39,11 +39,11 @@
 DEFAULT_EPSILON = 0.0001
 IMPLEMENTATION_OPTIONS = ['default', 'gunrock']
 
-TINY_DATASETS = ['../datasets/karate.csv']
+# TINY_DATASETS = ['../datasets/karate.csv']
```

Comment:
delete?

Reply:
There are some test that do not pass on the larger dataset, or get really slow.  Addressing those is step 2